crates/ursa-tracker/src/main.rs

```diff
@@ -105,12 +110,17 @@ async fn registration_handler(
 
 /// Prometheus HTTP Service Discovery
 async fn http_sd_handler(db: Extension<Arc<DB>>) -> (StatusCode, Json<Value>) {
+    let now = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap()
+        .as_millis() as u64;
     let services: Vec<PrometheusDiscoveryChunk> = db
         .iterator(IteratorMode::Start)
         .filter_map(|res| {
             if let Ok((_, v)) = res {
                 let node: Node = serde_json::from_slice(&v).unwrap();
-                if node.telemetry {
+                // only return registrations in the last 1 month
+                if now - node.last_registered < 2629800000 {
```

Comment:
Should we still check for `node.telemetry`, so `node.telemetry && now - node.last_registered < 2629800000` or is that not necessary?

Reply:
Don't think we really need to check `node.telemetry`, it's always set to true. It was from before when metrics and rpc were separated, but now their unified so a node will always expose their metrics